src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACLowTempRadiantVarFlow.cpp

```diff
@@ -84,6 +84,15 @@ namespace energyplus {
       return boost::none;
     }
 
+    boost::optional<HVACComponent> heatingCoil = modelObject.heatingCoil();
+    boost::optional<HVACComponent> coolingCoil = modelObject.coolingCoil();
+    if (!(heatingCoil.has_value() || coolingCoil.has_value())) {
+      LOG(Info,
+          modelObject.briefDescription() << " does not have either a Heating nor a Cooling Coil, it will not be translated");
+      return boost::none;
+    }
```

Comment:
Don't VT if the ZoneHVACLowTemp object doesn't have at least one heating or cooling coil.